rust/template/src/ovsdb_api.rs

```diff
@@ -128,7 +129,9 @@ fn dump_delta(
                 rel.iter()
                     .map(|(v, w)| {
                         assert!(*w == 1);
-                        record_into_insert_str(v.clone().into_record(), table_str)
+                        record_into_insert_str(v.clone().into_record(), table_str, &|t| {
+                            prog.get_table_original_name(t)
+                        })
```

Comment:
This is really neat, it makes me wonder if we shouldn't also store the *current* name so that we can improve errors somewhat